engine/access/rpc/engine.go

```diff
@@ -307,3 +326,27 @@ func (e *Engine) serveGRPCWebProxy() {
 		e.log.Err(err).Msg("failed to start the http proxy server")
 	}
 }
+
+// serveREST starts the HTTP REST server
+func (e *Engine) serveREST() {
```

Comment:
I do agree that REST API should not bring down the AN (and am changing the log level accordingly for the error) but making it an engine by itself might be an overkill since the backend that it uses is the common backed that all severs use. But if you feel strongly about it, I can issue it up and investigate making it an engine.